todo-list/src/components/App/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+
+const list=[
+  {
+    task:"item1",
+    id:123
+  },
+  {
+    task:"item2",
+    id:12
+  },
+  {
+    task:"item3",
+    id:1
+  }
+];
+
+const App = () => {
+  return (
+  list.map((item)=> {
+    return (
+      <div>
```

Comment:
@Arsalan74 It needs to have keys
```suggestion
      <div key={item.id}>
```

Reply:
Sorry I forgot to give elements their keys in feature1